tests/unit/lms/models/region_test.py

```diff
@@ -23,17 +27,34 @@ def test_from_request_raises_ValueError_for_bad_authorities(
         pyramid_request.registry.settings["h_authority"] = bad_authority
 
         with pytest.raises(ValueError):
-            Regions.from_request(pyramid_request)
+            Regions.set_region(pyramid_request)
+
+    @pytest.fixture
+    def current_region(self):
+        with mock.patch.object(Regions, "_current_region") as _current_region:
+            _current_region.__get__ = mock.Mock(return_value=None)
+            yield _current_region.__get__
```

Comment:
The global nature of this has the potential to cause some havok in testing, which is one down side. Here I've avoided the problem by ensuring these tests are isolated from the global object.